src/main/java/kitchenpos/application/MenuService.java

```diff
@@ -2,83 +2,67 @@
 
 import kitchenpos.dao.MenuDao;
 import kitchenpos.dao.MenuGroupDao;
-import kitchenpos.dao.MenuProductDao;
 import kitchenpos.dao.ProductDao;
 import kitchenpos.domain.Menu;
+import kitchenpos.domain.MenuGroup;
 import kitchenpos.domain.MenuProduct;
 import kitchenpos.domain.Product;
+import kitchenpos.dto.MenuProductRequest;
+import kitchenpos.dto.MenuRequest;
 import org.springframework.stereotype.Service;
 import org.springframework.transaction.annotation.Transactional;
 
-import java.math.BigDecimal;
-import java.util.ArrayList;
 import java.util.List;
-import java.util.Objects;
+import java.util.stream.Collectors;
 
 @Service
 public class MenuService {
     private final MenuDao menuDao;
     private final MenuGroupDao menuGroupDao;
-    private final MenuProductDao menuProductDao;
     private final ProductDao productDao;
 
     public MenuService(
             final MenuDao menuDao,
             final MenuGroupDao menuGroupDao,
-            final MenuProductDao menuProductDao,
             final ProductDao productDao
     ) {
         this.menuDao = menuDao;
         this.menuGroupDao = menuGroupDao;
-        this.menuProductDao = menuProductDao;
         this.productDao = productDao;
     }
 
     @Transactional
-    public Menu create(final Menu menu) {
-        final BigDecimal price = menu.getPrice();
+    public Menu create(final MenuRequest request) {
+        final MenuGroup menuGroup = getMenuGroup(request);
```

Comment:
`getMenuGroup(request);`

메서드에서 작성해주신 것 처럼 특정 `request` 를 매개변수로 받는게 더 좋다고 생각하실까요?
개인적으로 메서드마다 다르긴 하겠지만 위 메서드의 경우 필요한 값만 넘겨서 사용하는게 여러 요청에 대해 유연하게 사용해볼 수 있지 않을까 하는 생각이 조금 들긴하네요 :)
```suggestion
        final MenuGroup menuGroup = getMenuGroup(request.getMenuGroupId());
```

Reply:
아 저도 동의합니다! 기존 코드를 리펙토링하다 보니 복잡해져서 꼼꼼하지 못했던 것 같네요ㅠ 감사합니다!